### ISSUE TITLE
need some defines for NO_VGA

### ISSUE THREAD

**Issue by lowagner** [makapuf/bitbox#67]
need to add some define checking to compile with NO_VGA option, at least in these locations:

```
diff --git a/lib/emulator.c b/lib/emulator.c
index df6aaad..9b857fe 100644
--- a/lib/emulator.c
+++ b/lib/emulator.c
@@ -99,6 +99,7 @@ uint32_t time_left(void)
 }
 
 
+#ifndef NO_VGA
 extern uint16_t palette_flash[256];
 
 void __attribute__ ((weak, optimize("-O3"))) graph_line(void)
@@ -113,6 +114,7 @@ void __attribute__ ((weak, optimize("-O3"))) graph_line(void)
             draw_buffer[i] = palette_flash[drawbuf8[i]];
     }
 }
+#endif
 
 /* naive pixel conversion
 from 16bit bitbox pixel 0RRRRRGGGGGBBBBB
@@ -673,10 +675,12 @@ int main ( int argc, char** argv )
         // update time
         vga_frame++;
 
+        #ifndef NO_VGA
         if (scale==1)
             refresh_screen(screen);
         else
             refresh_screen2x(screen);
+        #endif
 
         SDL_Delay(time_left());
         next_time += slow ? TICK_INTERVAL*10:TICK_INTERVAL;
```

oh, and bitbox_main.c also requires it, since vga_frame isn't defined.
```
▸   ▸   #ifndef NO_VGA
▸   ▸   int oframe=vga_frame;
▸   ▸   while (oframe==vga_frame);
▸   ▸   #endif
```

**Comment by lowagner** [comment]
here's an example project:
https://github.com/lowagner/bitbox-modes/tree/master/noisebox

**Comment by lowagner** [comment]
fixed with a26ca5c6f0f45bc152ca86cad36c9373d9e4d15a
